### ISSUE TITLE
Dependabot not recreating or rebasing

### ISSUE THREAD

**Issue by kendallcarey** [dependabot/dependabot-core#10874]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

Mix

### Package manager version

Hex 2.1.1

### Language version

Elixir 1.14.5-otp-26

### dependabot.yml content

version: 2
updates:
  - package-ecosystem: "mix"
    directory: "/"
    schedule:
      interval: "daily"
    labels:
      - "dependencies"
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "daily"
    labels:
      - "dependencies"

### Updated dependency

httpoison from 1.0.0 to 2.2.1
scrivener_ecto from 2.7.0 to 3.1.0

### What you expected to see, versus what you actually saw

I have tried running `@dependabot rebase` and `@dependabot recreate` and I am consistently seeing `Dependabot tried to update this pull request, but something went wrong. We're looking into it, but in the meantime you can retry the update by commenting @dependabot rebase.` on both of the PRs. It has worked fine in the past.

**Comment by slonopotamus** [comment]
Same issue here: https://github.com/octobuild/octobuild/pull/123

**Comment by quaelin** [comment]
Yep, we are experiencing this, too.  Unfortunately, it means we are having to manually recreate PRs, so Dependabot is not really giving us much value for Elixir projects right now!

**Comment by barrieloydall** [comment]
yeah, for what seems like roughly a month we have had to manually recreate or fix PRs created by Dependabot when there is a conflict or the pr is out of date. `@dependabot rebase` and `@dependabot recreate` don't complete as reported in the issue 

Also don't think PR's are closing when a dependency has been updated by a different PR

`elixir 1.15.7-otp-25` 

help 😅

**Comment by jeffwidman** [comment]
We've shipped a bunch of fixes around this the past few months, I'm going to close. If you see it again (it can certainly happen, it's a complex system and we may have missed some edge cases), please file a new issue.
